frontend/src/App.tsx

```diff
@@ -51,7 +54,7 @@ export default function App () {
     return (
         <HashRouter basename={baseHref}>
             <ErrorBoundary>
-                { window.env.SYSTEM_BANNER?.text && <SystemBanner position='TOP' /> }
+                { applicationConfig.configuration.SystemBanner!.isEnabled && <SystemBanner position='TOP' /> }
```

Comment:
Is this guaranteed to be defined in the config? If yes, then we shouldn't need the `!`. If no, then we shouldn't add the `!`

From what I see in the shared model code. The ! should not be necessary and should be removed.

Reply:
Seems that it was set to optional in the type. I dont see why it has to be set to optional so I will change it to required. The default configuration just sets it to "False" so it should be fine

Reply:
It's optional because in the future when we have dynamic config for Projects, neither system banners nor project creation settings will be configurable at the project-level and hence we want to make those optional.

The `!` is fine in my view as the default app-wide config has something for this so it won't be undefined, and whenever the GET call succeeds and gives us the latest app config it will have this field as well.